UndertaleModTool/Editors/UndertaleEmbeddedTextureEditor.xaml.cs

```diff
@@ -55,7 +55,7 @@ private void Import_Click(object sender, RoutedEventArgs e)
                     
                     if ((width & (width - 1)) != 0 || (height & (height - 1)) != 0)
                     {
-                        MessageBox.Show("WARNING: texture page dimensions are not powers of 2. Sprite blurring is very likely in game.", "Unexpected texture dimensions", MessageBoxButton.OK, MessageBoxImage.Warning);
+                        MessageBox.Show(Application.Current.MainWindow as MainWindow, "WARNING: texture page dimensions are not powers of 2. Sprite blurring is very likely in game.", "Unexpected texture dimensions", MessageBoxButton.OK, MessageBoxImage.Warning);
```

Comment:
Can you create a property for this, so it doesn't have to get copypasted to every catch?